pkg/gateway/operations/putobject.go

```diff
@@ -309,7 +309,8 @@ func handlePut(w http.ResponseWriter, req *http.Request, o *PathOperation) {
 	// write metadata
 	metadata := amzMetaAsMetadata(req)
 	contentType := req.Header.Get("Content-Type")
-	err = o.finishUpload(req, blob.Checksum, blob.PhysicalAddress, blob.Size, true, metadata, contentType)
+	// BUG(ariels): Read MTime from upload!
```

Comment:
I think we still haven't decided if this should be fixed or not - anyway it's not related to the changes in this PR (let's track it in an issue instead?)

Reply:
Removed.